examples/crm/src/dashboard/LatestNotes.tsx

```diff
@@ -72,7 +72,11 @@ export const LatestNotes = () => {
                             key={`${note.type}_${note.id}`}
                             sx={{ marginBottom: 2 }}
                         >
-                            <Typography variant="body2" color="textSecondary">
+                            <Typography
+                                variant="body2"
+                                color="textSecondary"
+                                component="div"
```

Comment:
this is to avoid a warning on ReferenceField, because the Loader uses a div, and this is forbidden inside a `<p>`